WordPress/Classes/Utility/JetpackREST.m

```diff
@@ -0,0 +1,47 @@
+#import "JetpackREST.h"
+#import "AccountService.h"
+#import "BlogService.h"
+#import "ContextManager.h"
+
+static NSString * const JetpackRESTEnabledKey = @"JetpackRESTEnabled";
+
+@implementation JetpackREST
+
++ (BOOL)enabled
+{
+    return [[NSUserDefaults standardUserDefaults] boolForKey:JetpackRESTEnabledKey];
+}
+
++ (void)setEnabled:(BOOL)enabled withCompletion:(void (^)())completion
+{
+    NSUserDefaults *defaults = [NSUserDefaults standardUserDefaults];
```

Comment:
If the current flag was an instance variable I probably would have done that. I guess I instinctively avoided an extra call to `NSUserDefaults` to get the current value.